cafebabel/django_migrations.py

```diff
@@ -178,8 +178,8 @@ def create_article(old_article):
     tags = handle_groups(old_article['groups'])
     data = {
         'title': fields['title'],
-        'summary': article_fields['original_header'] or 'TODO',
-        'body': article_fields['body'] or 'TODO',
+        'summary': article_fields['original_header'] or '',
```

Comment:
When a summary was too long, it was ⬇️ moved as the  1️⃣st paragraph into the body and **bolded**.

Shall we ditch this case or treat it?

Reply:
It's not always the case :/